### PR TITLE
Added 'free' endpoint

### PR DESCRIPTION
**Pull request by marioncottard**

### CODE REVIEW

**Review by ggsdc — APPROVED** [review]

**Comment by ggsdc** [comment]
I'm not 100% convinced that we should have more than the get methods, no POST or PUT and see what we can do about filters

**Comment by marioncottard** [comment]
Okay, I'll delete them then. And I'll try and add the filters

File: cornflow-server/cornflow/shared/const.py [hunk 1]
```diff
@@ -137,4 +137,6 @@
     (SERVICE_ROLE, POST_ACTION, "deployed-dag"),
     (SERVICE_ROLE, POST_ACTION, "instance"),
     (SERVICE_ROLE, POST_ACTION, "execution"),
+    (SERVICE_ROLE, POST_ACTION, "tables"),
+    (SERVICE_ROLE, POST_ACTION, "tables-detail"),
 ]
```
Comment (ggsdc) [inline cornflow-server/cornflow/shared/const.py:142]: This permissions do not need to exist anymore

**Review by ggsdc — APPROVED** [review]

File: cornflow-server/cornflow/external_app/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .model import *
```
Comment (ggsdc) [inline cornflow-server/cornflow/external_app/__init__.py:1]: Why this change?

**Review by ggsdc — CHANGES_REQUESTED** [review]

File: cornflow-server/cornflow/external_app/__init__.py (hunk 2 above)
Comment (marioncottard) [reply cornflow-server/cornflow/external_app/__init__.py:1]: It was so that the import_models()  method in cornflow-server/cornflow/shared/utils.py, that imports all of the models for the TablesEndpoint, could find the models of external_app. Otherwise they weren't getting imported

Comment (ggsdc) [reply cornflow-server/cornflow/external_app/__init__.py:1]: Ok, but that part from external_app is getting moved, so we will have to see how to adapt it for the external_app later on. I will make a fast commit to delete ok?

Comment (marioncottard) [reply cornflow-server/cornflow/external_app/__init__.py]: Okay

**Review by ggsdc — APPROVED** [review]
